ibraheemosule/skoolizy
# za - updated settings page

# Why this PR?
Updated the settings page with another section 

# What does this PR do?
it adds another key section to the settings page which is the General Settings which contains the school name settings and also logo settings

# How to test your work?
-Navigate to the settings page
-Check for the School Name settings under the general settings section and once you click edit a modal should pop up to add your changes
-Check for the Logo settings under the general settings section and once you click edit a modal should pop up to add your changes

# Pages to see your changes?
/settings
